.gitignore

```diff
@@ -9,4 +9,6 @@ bookclub-bayes_rules.Rmd
 bookclub-bayes_rules.html
 bookclub-bayes_rules.knit.md
 bookclub-bayes_rules_files
+bookclub-bayes_rules_cache
+data/ch_11
```

Comment:
```suggestion
```

Reply:
I feel like this targeted ignore could lead to confusion. Since it isn't generated by the code or otherwise something that is likely to be there causing problems, I'd rather not ignore it in case a future cohort puts something there.